Framework/Backend/websocket/server.js

```diff
@@ -133,6 +133,42 @@ class WebSocket {
     client.on('error', (err) => this.logger.error(`Connection ${err.code}`));
   }
 
+  /**
+   * Make criteria more readable.
+   * This code is a close copy of InfoLogger/public/logFilter/LogFilter.js LN 101 toObject()
+   * @param {object} criteria - criteria to be minified
+   * @returns {object} minimal filter object
+   */
+  minifyCriteria(criteria) {
+    // Copy everything
+    const criterias = JSON.parse(JSON.stringify(criteria));
```

Comment:
Currently this function `minifyCriteria` is called form the WS server directly and there is no `try-catch` associated to it which means if the criteria is in a bad format it will crash the server.
I would recommend you add this function in a try catch and log in case of error that the payload criteria is not a correct format. We should not log the payload (as it is user input) as it might contain an injection attack